static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -170,5 +175,8 @@ define([
 
         // Mark the end of synchronous execution.
         userTiming.mark('App End');
+        robust.catchErrorsAndLog('ga-user-timing-enhanced-end', function () {
+            ga.trackPerformance('Javascript Load', 'enhancedEnd', 'Enhanced end parse time');
```

Comment:
I wondered that, probably value in both, though I don't want to send a shed load of requests and we can probably (need to test it out for sure) use [GA's calculated metrics](https://support.google.com/analytics/answer/6121409?hl=en) to create a 'duration' metric from end - start.